pkg/plugins/manager/signature/manifest_test.go

```diff
@@ -250,12 +251,12 @@ func TestCalculate(t *testing.T) {
 						filepath.Join(basePath, "plugin.json"):       {},
 						filepath.Join(basePath, "child/plugin.json"): {},
 					}, basePath),
-				})
+				}, publicKey)
 				require.NoError(t, err)
 				require.Equal(t, plugins.Signature{
-					Status:     plugins.SignatureValid,
-					Type:       plugins.GrafanaSignature,
-					SigningOrg: "Grafana Labs",
+					Status:     plugins.SignatureInvalid,
+					Type:       "",
+					SigningOrg: "",
```

Comment:
Same here as your comment on the other test case above, aren't we invalidating this test case if we do this?